tools/getrealdeps.rb

```diff
@@ -107,6 +107,9 @@ def main(pkg)
   pkgdeps = pkgdeps.map { |i| i.gsub(/glibc_build.*/, 'glibc') }.uniq
   pkgdeps = pkgdeps.map { |i| i.gsub(/glibc_lib.*/, 'glibc_lib') }.uniq.map(&:strip).reject(&:empty?)
 
+  # Leave early if we didn't find any dependencies.
+  return if pkgdeps.empty?
```

Comment:
Won't this exit if a package incorrectly has no dependencies listed?

That sounds like a bad idea?

Reply:
No this exits if we didn't find any runtime dependencies.